tslint.json

```diff
@@ -20,7 +20,7 @@
     "no-bitwise": true,
     "no-shadowed-variable": true,
     "no-unused-expression": true,
-    "no-unused-variable": true,
+    "no-unused-variable": [true, {"ignore-pattern": "ddescribe|iit|.*_$"}],
```

Comment:
I don't think we need it ignore `ddescribe` and `iit`. They shouldn't be imported anyway. @jelbourn any opinion?